marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubDeveloperTest.java

```diff
@@ -49,17 +51,33 @@ public void task9ConfigureBitemporal() throws IOException {
             "  }\n" +
             "}";
 
-        new TemporalAxesManager(userWithRoleBeingTestedClient, FINAL_DB).save(temporalAxis);
+        TemporalAxesManager mgr = new TemporalAxesManager(userWithRoleBeingTestedClient, FINAL_DB);
```

Comment:
For these tests that will require 10.0-3, can we adopt the same thing we did for mapping tests, where we require a certain nightly version or greater, or 10.0-3 or greater? Then @SameeraPriyathamTadikonda  doesn't need to do anything additional.

Reply:
Ok, will do